modules/nf-core/foldmason/createdb/main.nf

```diff
@@ -0,0 +1,47 @@
+process FOLDMASON_CREATEDB {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+            'oras://community.wave.seqera.io/library/foldmason:2.7bd21ed--a45f76ed12b391e6':
+            'community.wave.seqera.io/library/foldmason:2.7bd21ed--e7f739473ad6578d' }"
+
+    input:
+    tuple val(meta) , path(pdbs)
```

Comment:
To make it generic for `pdb`s and `mmcif`s I will suggest to call it structures instead
```suggestion
    tuple val(meta) , path(structures)
```